modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/utils/DeployVariableSchemaValidator.java

```diff
@@ -46,6 +61,14 @@ public static void validateDeployVariable(List<DeployVariable> deployVariables)
                 varNameSet.add(deployVariable.getName());
             }
         }
+        if (!missingExampleValueVariables.isEmpty()) {
+            String errorMessage = "The deploy variable configuration for "
+                    + DeployVariableKind.FIX_ENV.toValue() + " or " +
+                    DeployVariableKind.FIX_VARIABLE.toValue()
+                    + " kind has an empty example.";
```

Comment:
its better we have a more clear error message - 
Variable {variable_name} has type {variable_type} but a value for it is not provided.

Reply:
fixed.

Reply:
![image](https://github.com/user-attachments/assets/4e22944e-3995-4d72-9d50-d5f287c7af7d)